src/components/Faq.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import Accordion from '../components/Accordion';
+import { P, H2 } from './Typography';
+import styled from 'styled-components'
+import { CardLike } from './Common'
+
+const DetailFlex = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 100%;
+
+  @media (max-width: 1200px) {
+    display: block;
+  }
+`
+
+const DetailColumn = styled.ul`
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100%;
+  flex: 1;
+  margin: 0;
+  margin-right: 1em;
+  padding: 0;
+
+  & > li {
+    ${CardLike};
+    padding: 0.5em 1em;
+    margin-bottom: 1em;
+    list-style-type: none;
+  }
+`
+
+export default ({ faq }) => {
+  const categories = faq.reduce((q, a) => {
+    q[a.category] = [...q[a.category] || [], a]; // group by category
+    return q;
+  }, {});
+
+  function splitHalf(arr) {
+    const half = Math.ceil(arr.length / 2);
+    return [arr.splice(0, half), arr.splice(-half)];
+  }
+
+  function createFAQList(entries) {
+    return (
+      <DetailFlex>
+        {
+          splitHalf(entries).map(half =>
+            <DetailColumn>{half.map(singleEntry)}</DetailColumn>
+          )
+        }
+      </DetailFlex>
```

Comment:
is this to make two columns of FAQ boxes?

Reply:
yes

Reply:
Any reason why we can't use flexbox wrap instead?

Reply:
Then questions might get inadvertently shuffled around once you expand enough Qs, I tried this and the UX is kinda nasty